File: Python/Bubble sort.py
# List must contain the same datatype

list = [1, 4, 3, 5]

index = 0

def code():
    global var1, var2, list, index
    var1, var2 = list[index], list[index + 1]
    list[index], list[index + 1] = var2, var1

for x in range(len(list)):
    try:
        for i in range(len(list)):
            if type(list[0]) == str:
                if list[index][0] > list[index + 1][0]:
                    code()
            elif type(list[0]) == int:
                if list[index] > list[index + 1]:
                    code()
            
            index = index + 1
            
    except IndexError as error:
        pass
    finally:
        index = 0

print(list)

File: Javascript/Bubble sort.js
let list = [];

for (let x = 0; x < list.length; x++) {
    let index = 0;
    while ((index + 1) < list.length) {
        if (list[index] > list[index + 1]) {
            let var1 = list[index];
            let var2 = list[index + 1];
            list[index] = var2
            list[index + 1] = var1
        };
        index = index + 1
    };
};

console.log(list)